File: code-examples/csharp/products/get_variant.cs
var client = new RestClient("http://api.marketcloud.it/v0/products/:id/variants/:product_id");
var request = new RestRequest(Method.GET);
request.AddHeader("content-type", "application/json");
request.AddHeader("accept", "application/json");
request.AddHeader("authorization", "YourPublicKey:YourToken");
IRestResponse response = client.Execute(request);

File: code-examples/java/products/get_variant.java
HttpResponse<String> response = Unirest.get("http://api.marketcloud.it/v0/products/:id/variants/:product_id")
  .header("authorization", "YourPublicKey:YourToken")
  .header("accept", "application/json")
  .header("content-type", "application/json")
  .asString();